spring-integration-jmx/src/test/java/org/springframework/integration/jmx/configuration/EnableMBeanExportTests.java

```diff
@@ -142,9 +141,10 @@ public QueueChannel output() {
 			return new QueueChannel();
 		}
 
+		@SuppressWarnings("deprecation")
 		@Bean
-		public MetricsFactory myMetricsFactory() {
-			return new DefaultMetricsFactory();
+		public org.springframework.integration.support.management.MetricsFactory myMetricsFactory() {
```

Comment:
Can we consider to make this test without `myMetricsFactory`?
It is deprecated anyway, so maybe no reason to assert against at all...